src/Microsoft.Health.Dicom.Core/Messages/ExtendedQueryTag/GetExtendedQueryTagErrorsRequest.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using MediatR;
+
+namespace Microsoft.Health.Dicom.Core.Messages.ExtendedQueryTag
+{
+    public class GetExtendedQueryTagErrorsRequest : IRequest<GetExtendedQueryTagErrorsResponse>
+    {
+        public GetExtendedQueryTagErrorsRequest(string extendedQueryTagPath)
+        {
+            ExtendedQueryTagPath = extendedQueryTagPath;
+        }
+
+        /// <summary>
+        /// Path for the extended query tag error that is requested.
+        /// </summary>
+        public string ExtendedQueryTagPath { get; }
```

Comment:
nit: let's be consistent with the member naming regarding `TagPath`, Path`, or `ExtendedQueryTagPath` lol